### PR TITLE
[9.0][ADD] account_invoice_line_report: New module

### PR DESCRIPTION
**Pull request by carlosdauden**
This module adds invoice lines tree view based in account.invoice.report class.

@Tecnativa

### CODE REVIEW

File: account_invoice_line_report/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+============
+
+To install this module, just click the install button.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Account > Customers > Invoice Lines
```
Comment (pedrobaeza) [inline account_invoice_line_report/README.rst]: Why customers? Supplier invoice lines are not seen here?

File: account_invoice_line_report/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Account Invoice Line Report
+===========================
+
+This module create a new view to manage invoice lines information.
+
+Installation
```
Comment (pedrobaeza) [inline account_invoice_line_report/README.rst]: Remove this useless section

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by chienandalu — APPROVED** [review]
Nice addon. I tested the functionality on runbot (by the way, I had to install the accounting app :smile: )

**Review by pedrobaeza — APPROVED** [review]

File: account_invoice_line_report/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    'name': 'Account Invoice Line Report',
+    'summary': 'New view to manage invoice lines information',
+    'version': '9.0.1.0.0',
+    'category': 'Account',
+    'website': 'http://www.tecnativa.com',
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'depends': [
+        'account',
```
Comment (pedrobaeza) [inline account_invoice_line_report/__openerp__.py]: Change this to account_accountant for having the menu visible

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by luismontalba — APPROVED** [review]
Functionality tested on runbot

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and tested.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**